samples/word/07-custom-properties/read-write-custom-document-properties.yaml

```diff
@@ -11,45 +11,39 @@ script:
         $("#string").click(insertStringProperty);
         $("#read").click(readCustomDocumentProperties);
 
-        function insertNumericProperty() { 
+        function insertNumericProperty() {
             Word.run(function (context) {
                 context.document.properties.customProperties.add("Numeric Property", 1234);
                 return context.sync()
                     .then(function () {
-                        console.log("Property added");     
-                    })
-                    .catch(function (e) { 
-                        console.log(e.message);
+                        console.log("Property added");
                     })
+                    .catch(OfficeHelpers.Utilities.log);
             })
 
         }
 
-        function insertStringProperty() { 
+        function insertStringProperty() {
             Word.run(function (context) {
                 context.document.properties.customProperties.add("String Property", "Hello World!");
                 return context.sync()
                     .then(function () {
-                        console.log("Property added");     
-                    })
-                    .catch(function (e) { 
-                        console.log(e.message);
+                        console.log("Property added");
                     })
+                    .catch(OfficeHelpers.Utilities.log);
             })
         }
 
-        function readCustomDocumentProperties() { 
+        function readCustomDocumentProperties() {
             Word.run(function (context) {
                 var properties = context.document.properties.customProperties;
                 context.load(properties);
                 return context.sync()
                     .then(function () {
                         for (var i = 0; i < properties.items.length; i++)
-                            console.log("Property Name:" + properties.items[i].key + ";Type=" + properties.items[i].type +"; Property Value=" + properties.items[i].value);
-                    })
-                    .catch(function (e) { 
-                        console.log(e.message);
+                            console.log("Property Name:" + properties.items[i].key + ";Type=" + properties.items[i].type + "; Property Value=" + properties.items[i].value);
```

Comment:
Nit, add spaces.
You would also be far better off with a TypeScript template string https://basarat.gitbooks.io/typescript/content/docs/template-strings.html